exercises/darts/canonical-data.json

```diff
@@ -9,44 +9,62 @@
          "property": "score",
          "description": "A dart lands outside the target",
          "input": {
-           "x": 15.3,
-           "y": 13.2
+           "x": -9,
+           "y": 9
```

Comment:
This almost had me thinking it was inside the target.  I calculate the distance from the center to be  ~12.7 so checks out for this case.  👍